libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1673,3 +1673,23 @@ bool GCS_MAVLINK::telemetry_delayed(mavlink_channel_t _chan)
 }
 
 
+void GCS_MAVLINK::send_collision_all(const AP_Avoidance::Obstacle &threat, MAV_COLLISION_ACTION behaviour)
+{
+    for (uint8_t i=0; i<MAVLINK_COMM_NUM_BUFFERS; i++) {
+        if ((1U<<i) & mavlink_active) {
```

Comment:
That's an idiom used through the code when dealing with the mavlink backends.  My random "gcs singleton" PRs seek to remove that idiom, but until we do something I don't think we should do anything different in new code.